pretty_gpx/common/layout/base_vertical_layout.py

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/python3
 """Base Vertical Layout."""
+from collections.abc import Callable
 from dataclasses import dataclass
 from dataclasses import fields
+from typing import ClassVar
+from typing import Self
```

Comment:
```suggestion
from typing_extensions import Self
```

Reply:
@ThomasParistech Don't you think it is better to put python to 3.11 rather than using a backport feature ?

Reply:
`from typing_extensions import Self` is already used through the code

But you're right, let's upgrade python!

Reply:
Ah I have not seen that typing_extensions was already use